src/SongManager.h

```diff
@@ -179,6 +179,8 @@ class SongManager
 
 	void UpdatePopular();
 	void UpdateShuffled();	// re-shuffle songs and courses
+	void SetPreferredSongs(RString sPreferredSongs = "PreferredSongs.txt", bool bIsAbsolute = false);
+	void SetPreferredCourses(RString sPreferredCourses = "PreferredCourses.txt", bool bIsAbsolute = false);
```

Comment:
I think it would be cleaner to not have a default argument for the first argument.

Reply:
Yeah it seems not necessary. Removed.